src/s4c_the_church_rap_battle.tw2

```diff
@@ -0,0 +1,343 @@
+
+Moon Mystic approached Steve and I, holding both our hands, but I didn’t even wait for him. I raised my own hand and screamed, “YOUR CHAMPION [STAGE NAME]!” Moon Mystic laughed and nodded as [[the crowd stomped in unison|The Church - Mountain intro]].
+
+::The Church - Rap battle - Victory player second
+Moon Mystic approached Steve and I, holding both of our hands. I knew Steve and I had gone back and forth, but I wasn’t nervous.
+
+The crowd was like a wave, lifting me up. Moon Mystic raised my hand. “Your champion [STAGE NAME]!” As he said it, the crowd [[repeated it back and built a mighty chant|The Church - Mountain intro]].
+
+::The Church - Rap battle - Victory dog raps
+Steve's verse was about how basketball shorts are good for airing out sweaty nuts or something, but the dog crowd weren’t having it. Some of the larger dogs were trying to climb up on stage and snipping at Steve’s feet when he got too close to the edge.
+
+He lost his composure almost immediately as a weirdly large beagle bounded on stage howling and charging at him. He ran backstage, mic and all, screaming at an ear-splitting frequency.
+
+From backstage he announced into the mic, “For my own safety, I’ve decided to end tonight’s performance. We’ll try this again at a later date.”
+
+The crowd was like a banshee, booing and clapping and shrieking all at once. Moon Mystic approached me and raised my hand. [[“Your champion $stageName!”|The Church - Mountain intro]]
+
+::The Church - Mountain intro
+My eyes shone with the energy of victory. As I walked backstage, I could see Kiki jumping and waiting to hug me. As I reached her, I fell into her arms with exhaust and delight.
+
+We hung backstage for a while, recapping the greatest hits of the match and talking strategy, until I got called to the stage for [[the ceremony|The Church - Mountain ceremony]]. As Kiki squeezed me one last time, she said “Good luck on the mountain, Double L. You’re gonna be amazing.” Then I was gone, but her strength still flowed through me like pure caffeine.
+
+::The Church - Mountain ceremony
+The spotlight hit me as Moon Mystic’s assistants, decked in traditional neon green and black robes, lifted me up in an aluminum lawn chair. I bathed in the light and the crowd’s adoration as Moon Mystic read the rites over an open bottle of his namesake.
+
+(click: "an open bottle of his namesake")[$dissolve[“Take and drink. This is that good shit. May the haters take and eat unclean ass. You are the champ, and a ninja, and the most bomb-ass motherfucker forever and ever amen.” The crowd responded with a hearty amen. Moon Mystic dumped a beige powder into the bottle of Faygo and let it fizz a bit before offering it up to me.]]
```

Comment:
Let's add a comma after "forever and ever"

Reply:
Hmmm interesting. I'm used to saying this very quickly at a church with no pauses, but Randy Travis' song of the same name agrees with you. I added the comma.